children/schema.py

```diff
@@ -31,8 +30,8 @@
 
 
 class ChildNode(DjangoObjectType):
-    available_events = relay.ConnectionField(EventConnection)
-    past_events = relay.ConnectionField(EventConnection)
+    available_events = relay.ConnectionField("events.schema.EventConnection")
```

Comment:
This to avoid circular import in Event schema which import ChildNode from children schema